package.json

```
@@ -96,6 +96,7 @@
     "sequelize-cli": "5.5.1",
     "sequelize-temporal": "1.0.8",
     "showdown": "1.9.1",
+    "speakeasy": "^2.0.0",
```

You mentioned earlier you wanted to avoid this library. Have you changed your thoughts?

---

No, I tried otplib and was not able to make it work with Google Authenticator. I think it's still the most used authenticator library... the technology isn't really changing. So it should be fine.

---

```suggestion
    "speakeasy": "2.0.0",
```

We use fixed dependencies.